Client/src/Utils/stringUtils.js

```diff
@@ -15,3 +15,22 @@ export const capitalizeFirstLetter = (str) => {
 	}  
 	return str.charAt(0).toUpperCase() + str.slice(1);
 };
+
+/**
+ * Helper function to get first letter as a lower case string
+ * @param {string} str String whose first letter is to be lower cased
+ * @returns A string with first letter lower cased
+ */
+
+export const toLowerCaseFirstLetter = (str) => {	
+	if (str === null || str === undefined) {
+		  return "";
+	  }
+	  if (typeof str !== "string") {
+		  throw new TypeError("Input must be a string");
+	  }
+	  if (str.length === 0) {
+		  return "";
+	  }  
+	  return str.charAt(0).toLowerCase() + str.slice(1);
+  };
```

Comment:
What is the use case for this? Wouldn't just toLowerCase() be enough?

Reply:
toLowerCase() will change all to lowercases, this function is needed when only the first letter need to change to lower case, not the rest

Reply:
Yeah, but do we have a use case for that?